lms/djangoapps/student_account/views.py

```diff
@@ -301,6 +309,50 @@ def _external_auth_intercept(request, mode):
         return external_auth_register(request)
 
 
+def get_user_orders(user):
+    """Given a user, get the detail of all the orders from the Ecommerce service.
+
+    Arguments:
+        user (User): The user to authenticate as when requesting ecommerce.
+
+    Returns:
+        list of dict, representing orders returned by the Ecommerce service.
+    """
+    no_data = []
+    user_orders = []
+    allowed_course_modes = ['professional', 'verified', 'credit']
+    commerce_configuration = CommerceConfiguration.current()
+    # user_query = {'username': user.username}
```

Comment:
Actually this is depending on @tasawernawaz ecommerce PR so once that is merged I will uncomment this.